### ISSUE TITLE
Active uploads can not be stopped after clearing upload list

### ISSUE THREAD

**Issue by fajo-de** [pydio/cells#161]
Hi @cdujeu,

just ran into another issue with v.1.6.1.

Clearing the upload list also removes active (running) uploads. This causes issues, as the user is no longer be able to actually stop a running download unless he logs out and back in to Cells or reloads the browser tab.

I would suggest to have "Clear list" only remove stopped or failed downloads but leave running or halted downloads in the list to allow the user to deal with them.

Cheers,
Falk

**Comment by fajo-de** [comment]
Hi @cdujeu,

reloading the browser tab while having an active download now causes another issue - a stalled file. 

`Oct 16 07:20:30 xxxxxxx cells: {"level":"info","ts":"2019-10-16T07:20:30+02:00","logger":"pydio.grpc.tasks","msg":"Exec.DeleteNode : cannot find object in s3! Should it be removed from index?","OperationUuid":"delete-ebe31311-e552-4ef1-b7df-c55d46f6c4cb-2be43ba7","SpanRootUuid":"aba861ff-efd4-11e9-984e-782bcb00e5c6","SpanParentUuid":"aba861ff-efd4-11e9-984e-782bcb00e5c6","SpanUuid":"abbb3dc0-efd4-11e9-bce0-782bcb00e5c6","OperationUuid":"delete-ebe31311-e552-4ef1-b7df-c55d46f6c4cb-2be43ba7","NodePath":"xxxxxxxxxxxx"}`

Shouldn't the index be updated only after a successful transfer ? Re-indexing actually removes the stalled entry, but this has to be triggered manually and therefore is only available to the administrator.

**Comment by cdujeu** [comment]
Uploader was improved in master, will be in 2.1
